src/styles/main.scss

```
@@ -0,0 +1,57 @@
+%watch--features {
+  border: 1px black solid;
+  box-sizing: border-box;
+  border-radius: 50%;
+}
+
+%general--features {
+  position: absolute;
+  background-color: brown;
+}
+
+%hands--features {
+  @extend %general--features;
+  animation: timer 3600s infinite;
```

```suggestion
  animation: timer 3600s infinite linear;
```
```suggestion
  animation: timer 3600s infinite linear;  
``` // use 'linear' instead of default 'ease' (I know, this task is not on review, but I noticed this problem)))

---

nice, thks!